src/react/boosts/BoostCard.tsx

```diff
@@ -57,83 +57,81 @@ export class BoostCard extends React.Component<BoostCardProps> {
               return <>ERROR</>;
             }
 
-            if (data.listing) {
```

Comment:
Looks like I made a ton of changes in this file but all I did was remove the `if (data.listing)` check because it gets checked above, and so the rest is just whitespace changes.

Reply:
👍I think this was just for when we had bad test data anyway and I forgot to fully remove